applications/StructuralMechanicsApplication/custom_advanced_constitutive/constitutive_laws_integrators/generic_constitutive_law_integrator_kinematic_plasticity.h

```diff
@@ -505,7 +506,8 @@ class GenericConstitutiveLawIntegratorKinematicPlasticity
         double& rEquivalentStressThreshold,
         double& rSlope,
         ConstitutiveLaw::Parameters& rValues,
-        const double EquivalentPlasticStrain
+        const double EquivalentPlasticStrain,
+        const double CharacteristicLength
```

Comment:
why do you need the Lchar now

Reply:
It is needed in the _CurveFittingHardening_ curve type for computing the volumetric fracture energy g_f = G_f / Lchar. I am not sure if it is better to recalculated it or to bring it through the methods (it is computed outside the integrator)

Reply:
it is okkey then